pages/common/clang-format.md

```diff
@@ -13,12 +13,12 @@
 
 - Format a file using a predefined coding style:
 
-`clang-format --style={{LLVM|Google|Chromium|Mozilla|WebKit}} {{path/to/file}}`
+`clang-format --style={{LLVM|GNU|Google|Chromium|Microsoft|Mozilla|WebKit}} {{path/to/file}}`
```

Comment:
There are too many alternatives here. I think we should define the maximum limit of alternative placeholder count not to allow very long commands.

Reply:
@EmilySeville7cfg those changes could be nice as well but I don't think they are fully necessary, and I am definitely not the author of those. I don't file like it should be done in this PR or even by me, hence I am not sure why you _requested changes_ here?

Reply:
I've requested changes because another my comment requires them.

Reply:
@EmilySeville7cfg I guess you mention https://github.com/tldr-pages/tldr/pull/8493#discussion_r973734282.. My answer would be the same: my pull request is about fixing the list and I don't think I should be responsible I should do any other change in this PR.

Reply:
So I am going to resolve this conversation.